package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@nsftx/seven-gravity-gateway",
   "private": false,
-  "version": "3.0.3-beta.1",
+  "version": "3.0.3",
```

@dikaso This will be merged to `next`, can we preserve version number from next
```suggestion
  "version": "3.0.3-beta.1",
```